### PR TITLE
issue#80: modified the about page to it uses jekyll's louts and remov…

### PR DESCRIPTION
**Pull request by TabrisArg**
* [X]   created a layout for the about page.
* [X]   Added a logo.png file in the root folder to be shared by all the layouts
* [X]   modified about page to use layouts.
* [X]   removed the redundant code from the page.

### CODE REVIEW

File: _layouts/about.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{% include header.html %}
```
Comment (andiechu) [inline _layouts/about.html]: use base.html

File: _layouts/about.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<html lang="en">
```
Comment (andiechu) [inline _layouts/about.html]: {% include header.html %} ?

File: _layouts/about.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    <link http-equiv="Content-Type" content="text/html;charset=UTF-8" href="/vendor/fontawesome-free/css/all.min.css" rel="stylesheet">
+    <link http-equiv="Content-Type" content="text/html;charset=UTF-8" href="/vendor/simple-line-icons/css/simple-line-icons.css" rel="stylesheet" type="text/css">
+    <link rel="stylesheet" href="https://use.typekit.net/qpj4ugj.css" integrity="sha384-Ty/99WUHYq9Jc7Hvn1AuHmKI+C4AQE2Cbm4jY9kTsnDX3PdnXurTgegpu5qaQAUM" crossorigin="anonymous"> <!-- Montserrat font (bold, semiBold, Regular)-->
+    <link http-equiv="Content-Type" content="text/html;charset=UTF-8" href="/css/main.css" rel="stylesheet">
+</head>
+<body>
+{% include navbar.html %}
+{{content}}
+{% include footer.html %}
+</body>
```
Comment (andiechu) [inline _layouts/about.html]: there's already a </body> in footer.html

File: _layouts/projects.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<html lang="en">
```
Comment (andiechu) [inline _layouts/projects.html]: Same thing

File: _layouts/about.html (hunk 2 above)
Comment (TabrisArg) [reply _layouts/about.html]: Done

**Pull request merged**